docs/scripts/buildApi.ts

```diff
@@ -223,7 +223,7 @@ function extractEvents(project: TypeDoc.ProjectReflection, apisToGenerate) {
     });
   });
 
-  return events;
+  return events.sort((a, b) => (a.name > b.name ? 1 : -1));
```

Comment:
Maybe 
```suggestion
  return events.sort((a, b) => a.localeCompare(b));
```

Reply:
Their is indeed a change

```diff
     "name": "columnResizeStop",
     "description": "Fired when the user stops resizing a column. Called with an object <code>{ field: string }</code>."
   },
+  {
+    "name": "columnsChange",
+    "description": "Fired when the columns state is changed.\nCalled with an array of strings corresponding to the field names."
+  },
   {
     "name": "columnVisibilityChange",
     "description": "Fired when a column visibility changes. Called with a GridColumnVisibilityChangeParams object."
     "name": "columnWidthChange",
     "description": "Fired when the width of a column is changed. Called with a GridColumnResizeParams object."
   },
-  {
-    "name": "columnsChange",
-    "description": "Fired when the columns state is changed.\nCalled with an array of strings corresponding to the field names."
-  },
   { "name": "componentError", "description": "Fired when an exception is thrown in the grid." },
   {
     "name": "debouncedResize",
```

I'm updating :+1: